learningresources/tests/test_views.py

```diff
@@ -15,9 +15,16 @@
 class TestViews(LoreTestCase):
     """Hit each view."""
 
-    def test_status_get(self):
-        """Status page."""
-        resp = self.client.get("/lore/welcome", follow=True)
+    def test_get_home(self):
+        """Welcome page."""
+        resp = self.client.get("/home", follow=True)
+        self.assertTrue(resp.status_code == HTTP_OK)
+        body = resp.content.decode("utf-8")
+        self.assertTrue("Welcome" in body)
+
+    def test_get_root(self):
```

Comment:
It seems that this and the previous test should be combined to a single test, asserting that they both work and that they arrive at the same place.

Arriving at the same place _could_ be checked by checking that the body (or a hash of the body) is identical, but it's probably optional; just continuing to check for the 'Welcome' string should be enough. But combining the two tests should be done for clarity.